Remove payloads flatpak and rpmostree which are not supported by oe.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/anaconda.py | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/pyanaconda/anaconda.py b/pyanaconda/anaconda.py
index ad0ae605a..a47fefa5b 100644
--- a/pyanaconda/anaconda.py
+++ b/pyanaconda/anaconda.py
@@ -28,7 +28,6 @@ from pyanaconda.core import constants
 from pyanaconda.core.startup.dbus_launcher import AnacondaDBusLauncher
 from pyanaconda.modules.common.constants.services import PAYLOADS
 from pyanaconda.payload.source import SourceFactory, PayloadSourceTypeUnrecognized
-from pyanaconda.modules.payloads.payload.rpm_ostree.flatpak_manager import FlatpakManager
 from pyanaconda.ui.lib.addons import collect_addon_ui_paths
 
 from pyanaconda.anaconda_loggers import get_stdout_logger
@@ -82,14 +81,7 @@ class Anaconda(object):
             # Get the type of the DBus payload module if any.
             payload_type = self._get_dbus_payload_type()
 
-            if payload_type == PAYLOAD_TYPE_RPM_OSTREE:
-                if FlatpakManager.is_source_available():
-                    from pyanaconda.payload.rpmostreepayload import RPMOSTreePayloadWithFlatpaks
-                    klass = RPMOSTreePayloadWithFlatpaks
-                else:
-                    from pyanaconda.payload.rpmostreepayload import RPMOSTreePayload
-                    klass = RPMOSTreePayload
-            elif self.opts.liveinst:
+            if self.opts.liveinst:
                 from pyanaconda.payload.live import LiveOSPayload
                 klass = LiveOSPayload
             elif self.ksdata.liveimg.seen:
